multiversx_sdk_core/address.py

```diff
@@ -1,94 +1,106 @@
-from typing import Tuple
+import logging
+from typing import Protocol, Tuple
 
 from Cryptodome.Hash import keccak
 
 from multiversx_sdk_core import bech32
 from multiversx_sdk_core.constants import DEFAULT_HRP, METACHAIN_ID
 from multiversx_sdk_core.errors import ErrBadAddress, ErrBadPubkeyLength
-from multiversx_sdk_core.interfaces import IAddress, INonce
 
 SC_HEX_PUBKEY_PREFIX = "0" * 16
 PUBKEY_LENGTH = 32
 PUBKEY_STRING_LENGTH = PUBKEY_LENGTH * 2  # hex-encoded
 BECH32_LENGTH = 62
 
+logger = logging.getLogger("Address")
```

Comment:
in other file the logger name is with lower case, maybe keep the same format for name?

Reply:
renamed to `address`